chapter_multilayer-perceptrons/dropout.md

```diff
@@ -423,10 +429,17 @@ class DropoutMLP(d2l.Classifier):
         x = nn.relu(nn.Dense(self.num_hiddens_2)(x))
         x = nn.Dropout(self.dropout_2, deterministic=not self.training)(x)
         return nn.Dense(self.num_outputs)(x)
+```
```

Comment:
Note that if there's no text cell in between two code cells, both code cells will be merged during compilation.

Reply:
Nice catch, I'll just move the jax note below.